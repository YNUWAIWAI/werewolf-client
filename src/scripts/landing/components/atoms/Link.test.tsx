import * as React from 'react'
import {Link} from './Link'
import {shallow} from 'enzyme'

test('render', () => {
  const wrapper = shallow(
    <Link
      className="className"
      href="example.com"
    />
  )

  expect(wrapper.html()).toMatchSnapshot()
})
