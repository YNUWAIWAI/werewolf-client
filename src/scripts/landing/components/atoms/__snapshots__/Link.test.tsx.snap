// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `"<a class=\\"className\\" href=\\"example.com\\" rel=\\"noreferrer noopener\\" target=\\"_blank\\"></a>"`;
